home_style.css

```diff
@@ -0,0 +1,674 @@
+  font-size: 13px;
+  padding: 11px;
+  color: rgb(73, 73, 77);
+  margin: 10px 0;
+  width: 100%;
+  height: 10%;
+  line-height: 20px;
+}
+
+.moto2 {
+  font-family: cocogoose, sans-serif;
+  color: rgb(100, 100, 102);
+  font-weight: 600;
+  margin: 5px 0 0 0;
+}
+
+.credit {
+  font-family: 'Lato', sans-serif;
+  color: rgb(100, 100, 102);
+  font-weight: 600;
+}
+
+/* Courses Section */
+
+.main_courses {
+  height: 100%;
+  background-color: #272a31;
+  font-family: 'Lato', sans-serif;
+  background-image: url('./images/bricks.png');
+  background-blend-mode: multiply;
+  background-size: 3%;
+}
+
+.section_headline {
+  align-items: center;
+  justify-content: space-around;
+  height: 40px;
+}
+
+.section_title {
+  color: #fff;
+  font-weight: 400;
+  font-size: 20px;
+  font-family: 'Lato', sans-serif;
+}
+
+hr {
+  width: 10vw;
+  height: 1px;
+  background-color: #ec5242;
+  border: none;
+}
+
+.cards_container {
+  display: grid;
+  justify-items: center;
+  align-items: center;
+  grid-template-columns: 1fr;
+  grid-template-rows: repeat(5, 100px);
+  gap: 10px;
+  margin-top: 0;
+}
+
+.card {
+  background-color: rgba(211, 211, 211, 0.2);
+  backdrop-filter: blur(2px);
+  width: 95%;
+  height: 85px;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.course_icon {
+  width: 45px;
+  height: 46px;
+}
+
+.course_name {
+  color: #ec5242;
+  font-size: 15px;
+  width: 20%;
+  text-align: center;
+  overflow-wrap: break-word;
+}
+
+.course_info {
+  color: #fff;
+  font-weight: 400;
+  font-size: 12px;
+  width: 50%;
+  line-height: 15px;
+  font-family: 'Lato', sans-serif;
+}
+
+.course_link {
+  background-color: #ec5242;
+  text-decoration: none;
+  color: #fff;
+  height: 50px;
+  padding: 17px 40px;
+  margin-top: 20px;
+  margin-bottom: 30px;
+  font-family: 'Lato', sans-serif;
+}
+
+/* Teaching Team Section */
+
+.teaching_team {
+  background-color: #fff;
+  height: auto;
+  font-family: 'Lato', sans-serif;
+  margin-top: 16px;
+}
+
+.teaching_team .section_headline h2 {
+  color: #272a31;
+  font-weight: 600;
+}
+
+.the_team {
+  display: grid;
+  grid-template-rows: repeat(6, auto);
+  justify-items: center;
+  align-items: center;
+  margin: 5% 4%;
+}
+
+.teacher_info {
+  justify-content: space-around;
+  height: 100%;
+  width: 100%;
+  margin-top: 30px;
+}
+
+.img_container {
+  background-image: url('./images/hash-tag3.png');
+  background-repeat: no-repeat;
+  background-size: 50%;
+  width: 30%;
+  height: max-content;
+}
+
+.teacher_photo {
+  width: 60%;
+  height: 90%;
+  position: relative;
+  top: 8px;
+  left: 25px;
+}
+
+.info {
+  align-items: flex-start;
+  width: 65%;
+}
+
+.name {
+  font-family: 'Lato', sans-serif;
+  color: #272a31;
+  font-weight: 700;
+}
+
+.job {
+  font-size: 12px;
+  font-weight: 600;
+  color: #ec5242;
+  font-family: 'Lato', sans-serif;
+  font-style: italic;
+}
+
+.quotes {
+  margin-top: 9px;
+  font-size: 11px;
+  color: #272a31;
+  font-family: 'Lato', sans-serif;
```

Comment:
- [ ] [OPTIONAL] Very good step in writing these styles. I suggest taking advantage of the DRY method. You are using font-family Lato in all the following lines. It will be nice if you would just define a class with the font family and use that class in your HTML tags or the sections

Eg
```
.font-lato {
   font-family: 'Lato', sans-serif;
}

<h1 class="font-lato">My title </h1>
<blockquote class="font-lato">DRY method is amazing</blockquote>
```